python/ray/dag/compiled_dag_node.py

```diff
@@ -153,7 +153,14 @@ def __init__(self, idx: int, dag_node: "ray.dag.DAGNode"):
         self.dag_node = dag_node
 
         self.downstream_node_idxs: Dict[int, "ray.actor.ActorHandle"] = {}
-        self.output_channel = None
+        # Multiple return values are written to separate `output_channels`.
```

Comment:
Do we need multiple `output_channels`? Is it possible to use a single output channel where the channel itself knows which part of the output should be written to which downstream nodes?